lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -448,7 +448,7 @@ def get_object_ref(self, module, ib_obj_type, obj_filter, ib_spec):
             del ib_spec['restart_if_needed']
             ib_obj = self.get_object(ib_obj_type, obj_filter.copy(), return_fields=ib_spec.keys())
             # reinstate restart_if_needed key if it's set to true in play
```

Comment:
> if it's set to true in play

This is no longer what's going on, and I'm not entirely sure what _is_ happening. I would suggest changing the comment to clarify what is being checked here

Reply:
@Qalthos updated the comment and the PR.